dispatcher/tcp/tcp.go

```diff
@@ -110,9 +109,7 @@ func (d *TCP) handleConn(conn net.Conn) error {
 	if err != nil {
 		return fmt.Errorf("[tcp] %s <-> %s <-x-> %s handleConn dial error: %w", conn.RemoteAddr(), conn.LocalAddr(), server.Target, err)
 	}
-	_ = rc.(*net.TCPConn).SetKeepAlive(true)
 
-	_ = rc.SetDeadline(time.Now().Add(DefaultTimeout))
```

Comment:
Can you wrap `conn` and `rc` with a timeout mechanism before relay(conn, rc)?
conn.WriteDeadline and rc.ReadDeadline will be refresh before writing and reading.